src/App.js

```diff
@@ -51,21 +88,23 @@ const App = () => {
   }
 
   const resetGame = () => {
-    // Complete in Wave 4
+
```

Comment:
inside of reset game we want to set the player back to 1, set the winner to null, and call generateSquares inside of setSquares to reset the board.